install.py

```diff
@@ -30,6 +30,11 @@ def main():
     if s3_bucket:
         package_command_list.append('--s3-bucket')
         package_command_list.append(s3_bucket)
+        # Adding parameter to solve the limitation:
+        # Templates with a size greater than 51,200 bytes must be deployed via an S3 Bucket.
+        # The local template will be copied to that S3 bucket and then deployed.
+        deploy_command_list.append('--s3-bucket')
+        deploy_command_list.append(s3_bucket)
```

Comment:
Is this part of this or restore PR

Reply:
will now need it as part of this PR since the size threshold has crossed. once this is merged, will handle the conflicts in the #186

Reply:
ah ok!!